index.html

```diff
@@ -362,6 +362,15 @@ <h3>
           </aside>
         </section>
       </section>
+      <section>
+        <h3>
+          Permissions task source
+        </h3>
+        <p>
+          The <dfn class="export">permissions task source</dfn> is a [=task source=] used to
```

Comment:
Hmmm... maybe this shouldn't be exported... 

```suggestion
          The <dfn>permissions task source</dfn> is a [=task source=] used to
```

People should be using the user interaction task source for actual permissions checks... I'm worried people might use it wrong.